sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp

```diff
@@ -292,6 +299,39 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
     AZ_SECURITY_KEYVAULT_KEYS_DLLEXPORT static const KeyCurveName P521;
   };
 
+  /**
+   * @brief The encryption algorithm to use to protected the exported key material.
+   *
+   */
+  class KeyEncryptionAlgorithm final
+      : public Azure::Core::_internal::ExtendableEnumeration<KeyEncryptionAlgorithm> {
```

Comment:
@LarryOsterman, since you designed this type, is public inheritance from an  `_internal` type like this intended and acceptable?

Reply:
he made me use it , so it better be ok, otherwise i'll get upset as code copmplete has passed.